src/Http/Client/Response.php

```diff
@@ -465,11 +433,9 @@ protected function _getHeaders(): array
     }
 
     /**
-     * Provides magic __get() support.
-     *
      * @return string
      */
-    protected function _getBody(): string
+    protected function getBodyContents(): string
```

Comment:
`getBody` already exists in the laminas package, but it only returns the stream, not the contents.

Reply:
getContent() maybe?

Reply:
We already have a public method `getStringBody()` on that class, so this is just the protected method which gets used by `getJson()` and `getXml()` as well as the before mentioned `getStringBody()`